src/absil/ilx.fsi

```diff
@@ -40,20 +40,27 @@ type IlxUnionHasHelpers =
    | SpecialFSharpOptionHelpers 
    
 type IlxUnionRef = 
-    | IlxUnionRef of boxity: ILBoxity * ILTypeRef * IlxUnionAlternative[] * bool (* cudNullPermitted *)  * IlxUnionHasHelpers (* cudHasHelpers *)
+    | IlxUnionRef
+         of boxity: ILBoxity
+          * ILTypeRef
+          * IlxUnionAlternative[]
+          * bool (* cudNullPermitted *)
+          * IlxUnionHasHelpers (* cudHasHelpers *)
+          * bool (* cudVirtualTag *)
 
 type IlxUnionSpec = 
     | IlxUnionSpec of IlxUnionRef * ILGenericArgs
-    member EnclosingType : ILType
-    member GenericArgs : ILGenericArgs
-    member Alternatives : IlxUnionAlternative list
+    member EnclosingType     : ILType
```

Comment:
Please remove these whitespace normalizations for the purposes of the PR, so we can see the minimal diff.  Thanks